hibernate-core/src/test/java/org/hibernate/orm/test/cid/CompositeIdTest.java

```diff
@@ -156,8 +156,9 @@ public void testCompositeIds(SessionFactoryScope scope) {
 					statementInspector.clear();
 					Order o2 = new Order( c );
 					o2.setOrderDate( Calendar.getInstance() );
-					statementInspector.assertExecutedCount( 1 );
-					statementInspector.assertNumberOfOccurrenceInQuery( 0, "join", 1 );
+					statementInspector.assertExecutedCount( 2 );
```

Comment:
If this change is required it would mean that the `index` mapping in `Customer.hbm.xml` is ignored. The `Customer#orders` collection should be a `LIST`, not a `BAG` and hence this change should not be necessary.

Reply:
i checked an i 5.6 we execute a separate query for the lineitems, anywway i'll double check